### PR TITLE
Settings documentation updated

### LINKED ISSUE #191: Document the files in mifopay/settings package
**Summary:**

Document the files in mifopay/settings package: https://github.com/openMF/mobile-wallet/tree/master/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/settings

### PR DESCRIPTION
**Pull request by codepoet2017390**
Fixes #191 

Settings documentation updated

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by codepoet2017390** [comment]
@luckyman20  please review

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/settings/presenter/SettingsPresenter.java [hunk 1]
```diff
@@ -21,24 +23,38 @@
...
+     * @param localRepository instance of LocalRepository class
+     */
     @Inject
     public SettingsPresenter(UseCaseHandler useCaseHandler, LocalRepository localRepository) {
         mUseCaseHandler = useCaseHandler;
         mLocalRepository = localRepository;
     }
 
+    /**
+     *Attach view to presenter
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/settings/presenter/SettingsPresenter.java]:
```suggestion
     * Attach view to presenter
```

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/settings/presenter/SettingsPresenter.java [hunk 2]
```diff
@@ -21,24 +23,38 @@
...
+     * @param localRepository instance of LocalRepository class
+     */
     @Inject
     public SettingsPresenter(UseCaseHandler useCaseHandler, LocalRepository localRepository) {
         mUseCaseHandler = useCaseHandler;
         mLocalRepository = localRepository;
     }
 
+    /**
+     * Attach view to presenter
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/settings/presenter/SettingsPresenter.java]:
```suggestion
     * Attaches view to presenter
```

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/settings/presenter/SettingsPresenter.java [hunk 3]
```diff
@@ -21,24 +23,38 @@
...
+     * Executed when user clicks the logout button
+     */
     @Override
     public void logout() {
         mLocalRepository.getPreferencesHelper().clear();
         mSettingsView.startLoginActivity();
     }
 
+    /**
+     * Still under development
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/settings/presenter/SettingsPresenter.java]: Also mention what it is meant for.

**Comment by jsahil730** [comment]
@droidchef why wasn't this merged?

**Pull request closed without merging**